tests/helpers/generateFilesHelper-test.js

```diff
@@ -0,0 +1,6 @@
+// No tests needed ATM; all API Requests
```

Comment:
`generateFile` has some logic to create the `callBody` that could be tested, but ultimately we should move that logic out of the helper function.

Reply:
Would like to disregard this test at the moment per above comment and exclude from scope of this story.